ruby_audit.gemspec

```diff
@@ -17,13 +17,13 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/civisanalytics/ruby_audit'
   spec.license       = 'GPL-3.0-or-later'
 
-  spec.required_ruby_version = ['>= 2.5', '< 3.1']
+  spec.required_ruby_version = ['>= 2.5', '< 3.2']
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'bundler-audit', '~> 0.8.0'
+  spec.add_dependency 'bundler-audit', '~> 0.9.0'
```

Comment:
How come we need a bundler-audit update to 0.9.0?

I suppose we might as well increase the version to 0.9.0.1 to address the concern in the other PR.

Reply:
We need to upgrade because earlier versions of bundler-audit are not compatible with ruby 3.1. I can change it to a next patch version, but the pessimistic operator `~>` ensures that that version will get installed if we leave this as-is. However, if we change it to that patch version, then the highest installed version will be always be `0.9.0.x`, which is probably not what we want. More about this: https://thoughtbot.com/blog/rubys-pessimistic-operator

Reply:
Fair enough. Thanks for checking!

I think we can leave this as written.